templates/terraform/pre_create/access_approval_settings.go.erb

```diff
@@ -0,0 +1,15 @@
+updateMask := []string{}
+
+if d.Get("notification_emails") != nil {
```

Comment:
How doable would it be to generate these like we do for updatemasks in update? That would ensure that new fields get added here automatically.